src/lolex-src.js

```diff
@@ -743,6 +744,21 @@ function withGlobal(_global) {
                 delay: timeout
             });
         };
+        if (typeof _global.Promise !== "undefined" && utilPromisify) {
+            clock.setTimeout[
+                utilPromisify.custom
```

Comment:
A small comment/[link](https://nodejs.org/dist/latest-v10.x/docs/api/util.html#util_util_promisify_custom) explaining what `utilPromisify.custom` is could be useful for people reviewing the code ... ;-) 

```suggestion
                // Symbol used for custom overrides. See https://nodejs.org/dist/latest-v10.x/docs/api/util.html#util_util_promisify_custom
                utilPromisify.custom
```
~If this is indeed a `Symbol`, it would be useful to know when it was introduced wrt which Node versions we support, as this might be breaking (for some reason).~
We support Node LTS version 10 and 12 at the moment, and is this was introduced in Node 8, there are no breaking changes :)

Reply:
Lolex supports node 8 (please only drop it in a v6 release if you want to, even though it's EOL) https://github.com/sinonjs/lolex/blob/0e14fcac17e0cb49d5f7dd58227bec78b18703b3/.travis.yml#L5-L9

As you say this features is supported in Node 8 though, so no breaking change regardless